### PR TITLE
fix power select jumps

### PR DESCRIPTION
**Pull request by Romanior**

### CODE REVIEW

**Comment by Romanior** [comment]
ah just prevent the power-select shift the bottom content with 1px on focus.

**Comment by ZenCocoon** [comment]
@Romanior can you please update the changelog as well

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,10 @@
 ### master
 
+### 1.0.1 - 2017-02-16
```
Comment (ZenCocoon) [inline CHANGELOG.md]: please remove this line and keep under master until new version release

File: assets/stylesheets/bookingsync/_ember-power-select.scss [hunk 2]
```diff
@@ -4,26 +4,21 @@ $ember-power-select-disabled-background-color: #fff !default;
 $ember-power-select-multiple-selection-background-color: #fff !default;
 $ember-power-select-highlighted-background: $gray-lightest !default;
 $ember-power-select-selected-background: #fff !default;
-
```
Comment (ZenCocoon) [inline assets/stylesheets/bookingsync/_ember-power-select.scss]: Why to remove this, keeps concerns more clear I think

**Pull request merged**